app/models/spree/gateway/pay_pal_express.rb

```diff
@@ -15,16 +17,7 @@ def supports?(source)
     end
 
     def provider_class
-      ::PayPal::SDK::Merchant::API
-    end
-
-    def provider
-      ::PayPal::SDK.configure(
-        mode: preferred_server.present? ? preferred_server : "sandbox",
-        username: preferred_login,
-        password: preferred_password,
-        signature: preferred_signature)
-      provider_class.new
+      ActiveMerchant::Billing::PaypalExpressGateway
     end
 
     def auto_capture?
```

Comment:
I feel we should find a way to make this configurable, without `auto_capture?` set to true `solidus` is never gonna call our `purchase` method https://github.com/solidusio/solidus/blob/93aff014e269a7c26a521a5ba7524e2416cd7437/core/app/models/spree/payment/processing.rb#L26

Reply:
@apellizzn Good idea 👍  I think a custom preference setup like this should be enough:

```
preference :auto_capture, :boolean, default: true

def auto_capture?
  preferred_auto_capture
end
```

Reply:
Since we're extending the `Spree::Gateway` you can delete this override because the `auto_capture?` method is already managed here https://github.com/solidusio/solidus/blob/master/core/app/models/spree/payment_method.rb#L211

Reply:
@oeN thanks! Removed it. Admins can now choose this option from the admin UI:

<img width="260" alt="screen shot 2017-11-22 at 3 34 06 pm" src="https://user-images.githubusercontent.com/6490430/33133110-07577630-cf9c-11e7-8029-727b57a856c8.png">